app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -439,7 +439,7 @@ def parse_component_status(container_component_status)
         :condition => component_condition.type,
         :status    => component_condition.status,
         :message   => component_condition.message,
-        :error     => component_condition.error
+        :error     => (component_condition.error unless component_condition.error == "nil")
```

Comment:
@dkorn the `component_condition.error == "nil"` workaround belongs in the parser (so that we store a `null` value in the db). Also add a comment saying that it's a workaround and mention the openshift issue id (or link).